use-timescale/integrations/query-admin/psql.md

```diff
@@ -1,224 +1,241 @@
 ---
-title: Install the psql connection tool
-excerpt: How to install the psql client for PostgreSQL
+title: Connect to a Timescale Cloud service with psql 
+excerpt: Install the psql client for PostgreSQL and connect to your service 
 products: [cloud, mst, self_hosted]
 keywords: [connect, psql]
 ---
 
-# Install the psql connection tool
+# Connect with psql
 
-The `psql` command line tool is widely used for interacting with a PostgreSQL or
-Timescale instance, and it is available for all operating systems. Most of
-the instructions in the Timescale documentation assume you are using `psql`.
+You use `psql` command line tool to interact with your $SERVICE_LONG. Most procedures in the $COMPANY documentation assume you are using `psql`.
 
-Before you start, check that you don't already have `psql` installed. It is
-sometimes installed by default, depending on your operating system and other
-packages you have installed over time:
+To use `psql` to connect to your database, you need the connection details for your PostgreSQL server. Find those in the connection string generated during service creation. For more information, see [Connecting to Timescale][about-connecting].
 
-<Terminal>
-
-<tab label='Linux/macOS'>
-
-```bash
-psql --version
-```
-
-</tab>
-
-<tab label='Windows'>
-
-```powershell
-wmic
-/output:C:\list.txt product get name, version
-```
-
-</tab>
-
-</Terminal>
-
-## Install PostgreSQL package on macOS
-
-The `psql` tool is installed by default on macOS systems when you install
-PostgreSQL, and this is the most effective way to install the tool.
-On macOS you can use Homebrew or MacPorts to install the PostgreSQL package
-or just the `psql` tool.
-
-<Tabs label="Installing PostgreSQL package">
-
-<Tab title="Homebrew">
-
-<Procedure>
-
-### Installing PostgreSQL package using Homebrew
-
-1.  Install Homebrew, if you don't already have it:
+1. **Check for an existing installation**
 
+    `psql` is sometimes installed by default, depending on your operating system and other packages you have installed over time. 
+    
+    <Terminal>
+    
+    <tab label='Linux/MacOS'>
+    
     ```bash
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    psql --version
+    ```
+    
+    </tab>
+    
+    <tab label="Windows">
+    
+    ```powershell
+    wmic
+    /output:C:\list.txt product get name, version
     ```
+    
+    </tab>
+    
+    </Terminal>
```

Comment:
Add something in here saying that if psql is already installed, proceed directly to the connect section please.